File: code/C/raio.c
#include <stdio.h>

int main(void) {
    float pi = 3.14;
    int raio = 0;
    printf("Digite o valor do raio: ");
    scanf("%d", &raio);

    float resultado = pi * raio * raio;
    printf("O valor do raio é: %.2f\n", resultado);
    return 0;
}

File: code/Go/raio.go
package main

import "fmt"

func raio() {
	pi := 3.14
	var raio float64 = 0
	fmt.Println("Digite o valor do raio:")
	fmt.Scanf("%f", &raio)

	resultado := pi * raio * raio
	fmt.Println(resultado)
}

func main() {
	raio()
}
